src/elements/chip-group/chip-group/chip-group.visual.spec.ts

```diff
@@ -69,14 +70,14 @@ describe('sbb-chip-group', () => {
       it(
         `size=${size}`,
         visualDiffDefault.with(async (setup) => {
-          await setup.withFixture(template({ size: size }));
+          await setup.withFixture(template({ size: size as SbbFormFieldElement['size'] }));
```

Comment:
Nitpicky: Instead of casting two times the usages, it could also be casted on line 69 where it is created.